server/src/webserver.rs

```diff
@@ -927,7 +955,37 @@ mod tests {
         let llc = Data::new(compile_lexers(LANGUAGE_LEXER_ARR)); 
         let (file_contents, _) = save_source_as_string(actix_web::web::Json(test_source_file), llc);
         assert_eq!(file_contents, "# Test");
+    
+        
+        // <p><strong>Pass a block comment</strong></p>
+        let test_source_file = ClientSourceFile{
+            metadata: SourceFileMetadata {
+                mode: "python".to_string()
```

Comment:
Python doesn't have block comments -- use C/C++/etc. instead.

Reply:
made this correction